### ISSUE TITLE
Update mean and std

### ISSUE THREAD

**Issue by zhaohui-yang** [kuangliu/pytorch-groupnorm#2]
Hi,
  In PyTorch BatchNorm2d, there is a track_running_status parameter records the number of iterations, momentum used to merge previous mean and std with the current batch. I wonder this group normalization don't have this kind of parameters?
